python/get_python.sh

```diff
@@ -27,7 +27,17 @@ install_dependencies () {
         return $retVal
     fi
 
-    $DIR/pip.sh install -e $DIR/../scripts/o3de --no-deps --disable-pip-version-check  --no-warn-script-location
+    # if we're building a container app, create a package from o3de then install that to remove absolute paths to o3de scripts
+    if [ $O3DE_BUILDING_CONTAINER == 1 ]; then
+        pushd $DIR/../scripts/o3de/
+        $DIR/python.sh setup.py sdist
+        popd
+
+        $DIR/pip.sh install $DIR/../scripts/o3de/dist/o3de-1.0.0.tar.gz --no-deps --disable-pip-version-check --no-cache
```

Comment:
should we be pulling the filename from an ENV - assuming the `1.0.0` is based on the engine version, should we get that from `engine.json` or are they unrelated?

Reply:
They are unrelated, the is the version of the o3de scripts, we could version these the same as the engine version but 1.0.0 seems to be used across all platforms currently.